app/javascript/sdk/IFrameHelper.js

```diff
@@ -147,13 +156,16 @@ export const IFrameHelper = {
     loaded: message => {
       updateAuthCookie(message.config.authToken);
       window.$chatwoot.hasLoaded = true;
+      const campaignsSnoozedTill =
+        Cookies.get('cw_snooze_campaigns_till') || undefined;
```

Comment:
Why doe we need to do `|| undefined` here?

Reply:
I have removed it. It was done so we have the same value for absence of cookie on widget and SDK side.